docs/react-v9/contributing/dev-workflow.md

```diff
@@ -28,6 +28,8 @@ cd fluentui
 yarn
 ```
 
+Vanilla Windows users not working in WSL should run `yarn clean` and then `yarn build` weekly or when pulling from master breaks the environment.
```

Comment:
can you provide more details on this ? I'm not sure I understand why is this needed ?

Reply:
I think we can remove, I've only needed to run yarn build when there are linking or 
non-standard package linking changes in main (twice over the last year) - Ideally the information we provide here can be OS agnostic, although perhaps more necessary on some (windows) vs others.